packages/stac/lib/src/parsers/widgets/stac_border/stac_border.dart

```diff
@@ -9,10 +10,16 @@ part 'stac_border.g.dart';
 @freezed
 abstract class StacBorder with _$StacBorder {
   const factory StacBorder({
+    // Legacy properties for backward compatibility (Border.all)
```

Comment:
Let's not call it legacy, simply call it uniform border or border for all sides.